sources/packages/backend/workflow/src/workflow-definitions/fulltime-assessment-decisions.dmn

```diff
@@ -142,6 +142,47 @@
           <text>25</text>
         </outputEntry>
       </rule>
+      <rule id="DecisionRule_15szag4">
```

Comment:
Not be part of the PR. 
In the FT DMNs some of them start with `2021-2022`, `2022-2023`, `2023-2024`. Is there a particular reason for that? or it is just pending a cleanup.?

![image](https://github.com/user-attachments/assets/7d9cee72-b968-479a-830d-fc73c2b15233)

![image](https://github.com/user-attachments/assets/600cc706-3323-4dd5-9497-a383604c222e)

![image](https://github.com/user-attachments/assets/ecceea74-824f-44b6-a691-93da2648c9f3)

@andrewsignori-aot @guru-aot @andrepestana-aot ?

Reply:
In general, they do not need to be removed. 
If the point is that they will never going to be real production ones we can clean up (with biz approval).
For real production ones, we should keep them.